web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/DownloadImageControllerIT.java

```diff
@@ -0,0 +1,144 @@
+import edu.unc.lib.boxc.search.solr.models.ContentObjectSolrRecord;
+import edu.unc.lib.boxc.web.common.services.SolrQueryLayerService;
+import edu.unc.lib.boxc.web.services.rest.modify.AbstractAPIIT;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MvcResult;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlMatching;
+import static edu.unc.lib.boxc.auth.api.Permission.viewOriginal;
+import static edu.unc.lib.boxc.web.common.services.FedoraContentService.CONTENT_DISPOSITION;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import com.github.tomakehurst.wiremock.junit5.WireMockTest;
+
+/**
+ * @author snluong
+ */
+@ExtendWith(SpringExtension.class)
+@WebAppConfiguration
+@ContextConfiguration("/download-image-it-servlet.xml")
+
+@WireMockTest(httpPort = 46887)
+public class DownloadImageControllerIT extends AbstractAPIIT {
+    @Autowired
+    private AccessControlService accessControlService;
+    @Autowired
+    private SolrQueryLayerService solrSearchService;
+
+    @Test
+    public void testGetImageAtFullSize() throws Exception {
+        var pidString = makePid().getId();
+
+        stubFor(WireMock.get(urlMatching("/" + pidString + "/full/full/0/default.jpg"))
+                .willReturn(aResponse()
+                        .withStatus(HttpStatus.OK.value())
+                        .withBodyFile("bunny.jpg")
+                        .withHeader("Content-Type", "image/jpeg")));
+
+        MvcResult result = mvc.perform(get("/downloadImage/" + pidString + "/full"))
+                .andExpect(status().is2xxSuccessful())
+                .andReturn();
+
+        // Verify content was retrieved
+        MockHttpServletResponse response = result.getResponse();
+
+        assertEquals("image/jpeg", response.getContentType());
+        assertEquals("attachment; filename=image_full.jpg", response.getHeader(CONTENT_DISPOSITION));
```

Comment:
Is it possible to verify that the bunny image got returned as well? Or at least that a `InputStreamResource` was returned?